Arrowgene.Ddon.Shared/Entity/Structure/CDataCraftProductInfo.cs

```diff
@@ -0,0 +1,45 @@
+using Arrowgene.Buffers;
+
+namespace Arrowgene.Ddon.Shared.Entity.Structure
+{
+    public class CDataCraftProductInfo
```

Comment:
Add this to `Arrowgene.Ddon.Shared/Entity/EntitySerializer.cs`

Reply:
Done